test/Sentry.AspNetCore.Tests/SentryHttpMessageHandlerBuilderFilterTests.cs

```diff
@@ -74,11 +74,10 @@ public async Task Generated_client_sends_Sentry_trace_header_automatically()
                                 .GetRequiredService<IHttpClientFactory>()
                                 .CreateClient();
 
-                            await httpClient.GetAsync("https://example.com");
+                            await httpClient.GetAsync("https://fake.tld");
```

Comment:
flaky since https://example.com exists but not https://fake.tld.

Reply:
I don't follow. Why is it flaky?

The goal is to make sure we're not relying on a real DNS name and Internet to resolve it.

I ran these tests offline and the build failed due to such tests.